scripts/travis-deploy.sh

```diff
@@ -13,7 +13,7 @@ tag=$1
 # by an external user. Then we shouldn't push the docker image to docker hub.
 PUSH_FLAG=$([ -z "${CODALAB_DOCKER_USERNAME}" ] || echo "--push")
 
-python3 codalab_service.py build --version $tag --pull $PUSH_FLAG
+python3 codalab_service.py build --version v$tag --pull $PUSH_FLAG
```

Comment:
Our old Travis build step, for each release X.X.X, pushes a docker image with tag “X.X.X” and a docker image with tag “vX.X.X”

Our GitHub Actions workflow only pushes images with tag “vX.X.X” (this change makes that happen).

We only need “vX.X.X” images for deployment, though — the “X.X.X” images appear to be unused.